runtime/oti/cfr.h

```diff
@@ -942,6 +942,9 @@ typedef struct J9CfrClassFile {
 #define CFR_METHOD_EXT_HAS_METHOD_TYPE_ANNOTATIONS 0x01
 #define CFR_METHOD_EXT_HAS_CODE_TYPE_ANNOTATIONS 0x02
 #define CFR_METHOD_EXT_INVALID_CP_ENTRY 0x04
+#if JAVA_SPEC_VERSION >= 16
+#define CFR_METHOD_EXT_HAS_SCOPED_ANNOTATION 0x10
```

Comment:
Why is this not the `0x08`?

Reply:
@eric98zhang You can update this to use 0x8, I am working on updating mine PR to attempt to reuse the `J9AccMethodFrameIteratorSkip` flag instead of using a new extended modifier

Reply:
You tagged the wrong user